.gitignore

```diff
@@ -6,7 +6,7 @@ Gemfile.lock
 downloads
 
 # node modules
-node_modules
+# node_modules
```

Comment:
@CITguy do you think it's better to just commit the toolkit build as a static file set since we're not using anything else for Node on this site, or should I have TravisCI download it on the fly? If I do option 2, I need to update the Dockerfile for previews so the library is pulled down.

Reply:
either way should result in the same assets, so whatever is easiest for y'all

Reply:
so, rather than blanketing _all_ of `node_modules`, maybe we should update to include patterns to match things we actually need...

```
node_modules
!node_modules/@webcomponents/*.js
!node_modules/helix-ui/dist
```